doozer/doozerlib/osbs2_builder.py

```diff
@@ -135,6 +135,10 @@ async def build(self, image: "image.ImageMetadata", profile: Dict, retries: int
 
         if build_info:
             logger.info("Successfully built image %s; task: %s ; nvr: %s ; build record: %s ", image.name, task_url, build_info["nvr"], build_url)
+
+            # Trigger SAST Scanning with OpenScanHub
+            logger.info("Firing off SAST scan")
+            exectools.fire_and_forget(self._runtime.working_dir, f"osh-cli mock-build --config=cspodman --brew-build {build_info['nvr']} --email asdas@redhat.com")
```

Comment:
there is also a `--nowait` switch to `mock-build`, which may or may not be handy.

Reply:
Yup we need that.